azure-pipelines.yml

```diff
@@ -1,7 +1,54 @@
 jobs:
-- job: Docker
+- job: Modern
```

Comment:
to be honest, I don't like naming like `Modern`.
okay, today it's modern for sure, but in a year or several it may easily became outdated, and will only introduce a confusion.
I'd say let's use some non-timing naming, such as `same-glibc`, `xenial-based`, or whatever.

Reply:
Done. Renamed to "Xenial".